internal/kubernetes/signer/README.md

```diff
@@ -0,0 +1 @@
+Copied from: https://github.com/kubernetes/kubernetes/tree/v1.18.0-beta.2/pkg/controller/certificates/signer
```

Comment:
:+1: thanks for the README here :) in future it may be worth extracting some of these stuff out into our own package, but this works!

Reply:
It's also worth noting this PR which allows configurable intermediate certs to be included as part of the Certificate: https://github.com/kubernetes/kubernetes/pull/88741